.github/workflows/pull-request-next.yml

```diff
@@ -96,8 +97,12 @@ jobs:
 
       - run: yarn test:visual
         env:
-          APPLITOOLS_BATCH_ID: ${{ github.event.pull_request.head.sha }}
-          EYES_API_KEY: ${{ secrets.EYES_API_KEY }}
+          VRT_APIURL: ${{ secrets.VRT_APIURL }}
```

Comment:
Compared with yout setup here we have some additional env variables:

```
CHOKIDAR_USEPOLLING: 1
CYPRESS_PORT: 7070
GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Additionally, can you check if some of your repository-level secrets are missing?

![image](https://github.com/Typeform/embed/assets/96059844/a3db4b66-c66b-4d1c-8bcb-02565f3d405f)

Reply:
I think I understand the issue your facing; we're not running in a self-hosted GitHub runner because this repo is public. So we'll have to find a workaround for this.